### PR TITLE
Add GitHub Actions workflow 

### PR DESCRIPTION
**Pull request by zemdom**
Workflow for scanning license headers and static analysis of Ansible playbooks.

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/benchmarking/2)

### CODE REVIEW

File: utils/check_license/check-headers.sh [hunk 1]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2016-2020, Intel Corporation
```
Comment (mmatuszek1) [inline utils/check_license/check-headers.sh]: Copyright 2021

File: utils/check_license/file-exceptions.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2016-2019, Intel Corporation
```
Comment (mmatuszek1) [inline utils/check_license/file-exceptions.sh]: Copyright 2021

**Review by mmatuszek1 — APPROVED** [review]

**Review by hsoltysi — CHANGES_REQUESTED** [review]
> Reviewed 1 of 3 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/pmem/benchmarking/2)* status: all files reviewed, 2 unresolved discussions (waiting on @mmatuszek1 and @zemdom)

___
*[utils/check_license/check-headers.sh, line 3 at r1](https://reviewable.io/reviews/pmem/benchmarking/2#-MaTMXQlQjXV8sp08Iod-r1-3:-MaTW8DB5SnKi9oED6xI:b-j9huyq) ([raw file](https://github.com/pmem/benchmarking/blob/866df194aab23631a33727155cbfe7d5061e7519/utils/check_license/check-headers.sh#L3)):*
<details><summary><i>Previously, mmatuszek1 wrote…</i></summary><blockquote>

Copyright 2021
</blockquote></details>

It seems we should stay with original file dates, when not altering file. We'd prefer to get newer version of this checker from:
 https://github.com/pmem/pmemkv/blob/master/utils/check_license/check-headers.sh

**Review by hsoltysi — APPROVED** [review]
> Reviewed 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/benchmarking/2)* status: all files reviewed, 2 unresolved discussions (waiting on @mmatuszek1 and @zemdom)

**Review by hsoltysi — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/benchmarking/2)* status: all files reviewed, 2 unresolved discussions (waiting on @mmatuszek1 and @zemdom)

**Pull request merged**